web-components/graphrag-widget/src/styles.css

```
@@ -0,0 +1,539 @@
+/* All styles scoped under .fdb-widget to avoid leaking into host page */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap');
```

Stylelint (17.7.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap'" (import-notation)

(import-notation)